rust/gitxetcore/src/config/util.rs

```diff
@@ -126,6 +127,46 @@ impl<T: Sized> OptionHelpers<T> for Option<T> {
     }
 }
 
+/// Returns a sanitized version of the command used to invoke the current process, with absolute paths stripped
+/// of the current working directory
+pub fn get_sanitized_invocation_command(strip_program_path: bool) -> String {
+    let mut args = std::env::args();
+    let invocation_prog = args
+        .next()
+        .map(|prog| {
+            if strip_program_path {
+                Path::new(&prog)
+                    .file_name()
+                    .unwrap_or_default()
+                    .to_str()
+                    .unwrap_or_default()
+                    .to_owned()
+            } else {
+                prog
+            }
+        })
+        .unwrap_or_default();
+
+    let cwd = std::env::current_dir()
+        .unwrap_or_default()
+        .to_str()
+        .unwrap_or_default()
+        .to_owned();
+
+    let subcommand: String = args
+        .map(|ee| {
+            let e = ee.strip_prefix(&cwd).unwrap_or(&ee);
```

Comment:
May want to use Path::strip_prefix instead because str::strip_prefix leaves the forward slash in the result at the beginning.